notebooks/llm-question-answering/llm-question-answering.ipynb

```diff
@@ -9,14 +9,14 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        # num_current_toks = len(tokenizer.encode(new_gen_text))
</pre><p>why this disabled?</p><p>You can use Streamer for calculation time between previous and current token, it can be used for estimation pipeline latency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2062/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
restored